nexus_motion_planner/CMakeLists.txt

```diff
@@ -72,6 +77,8 @@ install(
 )
 
 if(BUILD_TESTING)
+  find_package(ament_cmake_gtest REQUIRED)
```

Comment:
Doesn't seem to be used?

Reply:
3e2bb014750b8a350cb747804d6f43fae188d814